benchmark/matmul.cpp

```diff
@@ -166,21 +166,13 @@ static void SingleMatmulBase(
         "Shared memory bank conflict not removed.");
   }
 
+  std::vector<c10::IValue> aten_inputs({inputs.first, inputs.second});
+
   // Warm up run
-  auto outputs = fe.runFusion({inputs.first, inputs.second});
-  fe.setMeasureKernelTimeFlag(true);
+  auto outputs = fe.runFusion(aten_inputs);
   checkMatch(expected_output, outputs.at(0).to(at::kDouble), input_mnk.at(2));
 
-  // Sync everything up before we start
-  for (auto _ : benchmark_state) {
-    clearL2Cache();
-    auto outputs = fe.runFusion({inputs.first, inputs.second});
-    checkMatch(expected_output, outputs.at(0).to(at::kDouble), input_mnk.at(2));
```

Comment:
@drzejan2 Is it necessary to verify the results of every iteration? Assuming the inputs are the same, is the check above the loop enough?

Reply:
You are right, for the same set of inputs we should check the results only once so lets remove the check in every iteration. Thanks!

The only scenarios that come to my mind where we may want to check values after each an every execution are when:
- there is some global object on device which can be in dirty state and this may cause UB,
- we want to verify the impact of ECC disabled,
but this is not the case for us.